pancancer_evaluation/utilities/data_utilities.py

```diff
@@ -258,9 +258,17 @@ def split_by_cancer_type(rnaseq_df,
     """Split expression data into train and test sets.
 
     The test set will contain data from a single cancer type. The train set
-    will contain either the remaining data from that cancer type, or the
-    remaining data from that cancer type and data from all other cancer types
-    in the dataset.
+    will contain one of the following:
+        - only the remaining data from the same cancer type
```

Comment:
This is a good explanation of what's going on. Is there an argument for making it one variable with three levels? ie just_cancer_type, just_other_type, all?

Reply:
Yeah, that would make more sense! In the scripts I'm running directly (e.g. `run_cancer_type_classification.py`) it makes sense to have the `pancancer_only` argument (although that's not a great name), but for this function I think you're right, it would be clearer to have one variable with three levels.